docs/how-tos/providing-images-for-img.rst

```diff
@@ -5,25 +5,14 @@ Providing images for ``<img>``
 
 :author: `pkel <https://github.com/pkel>`_
 
-In this how-to, you will learn how to create an endpoint for providing images to HTML :code:`<img>` tags without client side JavaScript.
-The resulting HTML might look like this:
-
-.. code-block:: html
-
-   <img src="http://host/files/42/cats.jpeg" alt="Cute Kittens"/>
-
-In fact, the presented technique is suitable for providing not only images, but arbitrary files.
+In this how-to, you will learn how to create an endpoint for providing images to HTML :code:`<img>` tags without client side JavaScript. In fact, the presented technique is suitable for providing not only images, but arbitrary files.
```

Comment:
I'd like to add a warning for this how-to, what I mentioned in https://github.com/PostgREST/postgrest-docs/issues/327#issue-626719955.

> Also maybe add a note with a link to https://wiki.postgresql.org/wiki/BinaryFilesInDB
> And mention CDN as an alternative for scaling, see the discussion on https://github.com/PostgREST/postgrest/issues/1077.

Maybe the CDN part can be left out, it's not only about scaling in throughput but pg WALs(backups, replicas) also get bloated with big files.

Reply:
Just linking to the discussion would be good though